src/Controller/WishlistConfigurationAdminController.php

```diff
@@ -67,12 +76,10 @@ public function statisticsAction()
 
     public function resetStatisticsCacheAction()
     {
-        $result = $this->get('doctrine.cache.provider')->deleteMultiple([
-            BaseGridDataFactory::CACHE_KEY_STATS_ALL_TIME,
-            BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_DAY,
-            BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_MONTH,
-            BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_YEAR,
-        ]);
+        $result = $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_ALL_TIME);
+        $result &= $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_DAY);
+        $result &= $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_MONTH);
+        $result &= $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_YEAR);
```

Comment:
`&=` is a bitwise operator and will return an integer, not a boolean.

It would be better to have:

```suggestion
        $result = $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_ALL_TIME)
            && $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_DAY)
            && $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_MONTH)
            && $this->cache->delete(BaseGridDataFactory::CACHE_KEY_STATS_CURRENT_YEAR);
```